Documentation/dl_build.md

```diff
@@ -18,7 +18,6 @@ To build `etcd` from the `master` branch without a `GOPATH` using the official `
 $ git clone https://github.com/coreos/etcd.git
 $ cd etcd
 $ ./build
-$ ./bin/etcd
```

Comment:
The other examples include running etcd; either those lines should be removed for consistency or the `To build` bit can be replaced with `To build and run`. I'm leaning toward build and run because it's more obvious where the build drops the binary.